src/vg/civcraft/mc/prisonpearl/listener/PlayerListener.java

```diff
@@ -429,7 +449,7 @@ else if(event.getAction() == InventoryAction.PLACE_ALL
 				
 				InventoryHolder holder = !clickedTop ? event.getView().getTopInventory().getHolder() : event.getView().getBottomInventory().getHolder();
 				if (holder==null){
-					Player player=pearl.getHolderPlayer();
+					pearl.getHolderPlayer();
```

Comment:
uh... isn't this supposed to be assigned to a thing? like, holder = pearl.getHolderPlayer() since it falls through to an additional check below? same QQ for line 423 above.